gitlab/tests/common.py

```diff
@@ -151,3 +159,23 @@
         }
     ],
 }
+
+
+def assert_check(aggregator, metrics):
```

Comment:
Moved from https://github.com/DataDog/integrations-core/pull/7494/files#diff-5156a8a0dd99559d9073e00a8ff55993L23-L39 and included `aggregator.assert_all_metrics_covered()` at the end

Reply:
> and included aggregator.assert_all_metrics_covered() at the end

Note: Looks like we're not calling `assert_all_metrics_covered()` in `test_e2e.py` yet. (I assume that's expected because we didn't do it in the previous `test_integration_e2e.py` file, but just wanted to point that out.)

Reply:
Yes, I tried adding that but resulted in flakeyness